pkgs/by-name/lo/log4brains/package.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  yarnInstallHook,
+  nodejs,
+  yarn,
+  moreutils,
+  jq,
+  makeBinaryWrapper,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "log4brains";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "thomvaill";
+    repo = "log4brains";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-2EAETbICK3XSjAEoLV0KP2xeOYlw8qgctit+shMp5Qs=";
+  };
+
+  yarnOfflineCache = fetchYarnDeps {
+    yarnLock = finalAttrs.src + "/yarn.lock";
+    hash = "sha256-HHiWlOYwR+PhfpQlUfuTXUiQ+6w1HATGlmflQvqdNlg=";
+  };
+
+  # generated from https://codeberg.org/tropf/log4brains
+  patches = [
+    ./0001-replace-version-check-using-package.json.patch
+    ./0002-move-nextjs-build-into-temporary-directory.patch
+  ];
```

Comment:
You can also use `fetchpatch2` for these patches. Something like:
```
fetchpatch2 {
  url = "https://codeberg.org/tropf/log4brains/commit/8c7af30fe7377235037c1a385f484f201ecfe063.patch"
  hash = "";
}
```
If you are going to maintain patches on your fork.